styles.css

```diff
@@ -124,4 +124,12 @@ a p {
 
 #footer:hover p{
     bottom:0;
+}
+
+/*Set width for an 8 inch tablet and below. Phillip Andrews*/
+@media (width <= 768px) {
+	#footer img {
+		width: 30%;
+		margin-top: 207px;
```

Comment:
can you tell me, what made you think altering the margin part will fix in any way?
the code looks good.
will merge
thanks